.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -34,11 +34,14 @@ merge of your pull request!
 
 <!-- to check an item, place an "x" in the box like so: "- [x] Documentation" -->
 
-- [ ] Documentation added to the
-      [docs site](https://github.com/alexkrolick/testing-library-docs)
-- [ ] I've prepared a PR for types targetting https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/testing-library__dom
+- [ ] Documentation added to the [docs site][docs]
+- [ ] I've prepared a PR for types targeting [DefinitelyTyped][types]
 - [ ] Tests
 - [ ] Ready to be merged
       <!-- In your opinion, is this ready to be merged as soon as it's reviewed? -->
 
 <!-- feel free to add additional comments -->
+
+[docs]: https://github.com/testing-library/testing-library-docs
+[types]:
+  https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/testing-library__dom
```

Comment:
Oh, actually, these should be where they were before because people don't look at this when it's rendered, they see it when they fill it out, so I want those where they were. Thanks!